content/md/what-is/what-about-measurements.md

```diff
@@ -0,0 +1,46 @@
+---
+id: what-about-measurements
+themes: first-steps
+title: What about **measurements**?
+popular: false
+ee-only: false
+related: manage-your-measurements
+---
+
+# Introduction to the measurements
+
+If you want to **store your product information with measurement**, i.e. weight, length or frequency inside your PIM, you will need [measurement families](#what-is-a-measurement-family) .
+
+The measurements will be really helpful when you are exporting products for a given channel and you want these measurement attributes to be converted into the units you set in the [given channel](manage-your-channels.html#create-a-channel).
+
+A measurement attribute is a single-line field composed of a first field containing a quantity and a second field containing a measurement unit.
+
+Below is an example of a `Weight` [measurement attribute](manage-your-attributes.html#create-an-attribute) with a quantity of `533` for the `Gram` unit.
+
+![Example of a measurement attribute](../img/Settings_Measurement_Attribute.png)
+
+# What is a measurement family?
+
+A measurement family is used to group the units concerning the same kind of quantity for a product measurement, i.e. a weight, a length, a frequency, etc.
+
+For example, the `Weight` measurement family allows you to store the weight of products in grams, kilograms, livres, pounds, etc.
```

Comment:
'Livres' is a French word! You could use ounce instead?

Reply:
OK